internal/witness/cmd/feeder/main.go

```diff
@@ -38,11 +38,28 @@ import (
 )
 
 var (
-	logList  = flag.String("log_list_url", "https://www.gstatic.com/ct/log_list/v3/log_list.json", "The location of the log list")
-	witness  = flag.String("witness_url", "", "The endpoint of the witness HTTP API")
-	interval = flag.Duration("poll", 10*time.Second, "How quickly to poll the log to get updates")
+	logList     = flag.String("log_list_url", "https://www.gstatic.com/ct/log_list/v3/log_list.json", "The location of the log list")
+	witness     = flag.String("witness_url", "", "The endpoint of the witness HTTP API")
+	interval    = flag.Duration("poll", 10*time.Second, "How quickly to poll the log to get updates")
+	getByScheme = map[string]func(*url.URL) ([]byte, error){
```

Comment:
Perhaps consider moving this map down to the bottom of the file, along with the `read.*` funcs - in general it seems the guidance for go files is to put the most important/salient bits up the top and the incidental rando support bits at the bottom.

Reply:
Good to know!  That's done now.